scripts/split_test_output.py

```diff
@@ -8,18 +8,19 @@
 # Usage: `cat <file> | ./scripts/split_test_output.py`
 
 import sys
-import os
 import re
 
-id_regex = re.compile("id: (\d+)")
+
+id_regex = re.compile("id: (\\d+)")
 ansi_escape = re.compile(r'(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]')
 
-def split_input(input):
+
+def split_input(input_stream):
```

Comment:
Thoughts on adding a type to `input_stream` here?

Reply:
input is a built-in python function, it is not correct to use built-in python keywords as variable names

Reply:
In this case, I mean adding a type as `input_stream: SomeTypeHere`, not the name itself.

Reply:
Ah sorry, fixed it.